src/api/types.ts

```diff
@@ -147,18 +161,7 @@ export interface Cluster {
   /**
    * Status of the OpenShift cluster.
    */
-  status:
```

Comment:
Don't do this, api/types.ts file is autogenerated from swagger

Reply:
done.